epoch1/cuda/ee_mumu/SubProcesses/P1_Sigma_sm_epem_mupmum/check.cc

```diff
@@ -143,6 +152,12 @@ int main(int argc, char **argv)
     return usage(argv[0]);
   }
 
+  std::ofstream dumpFile;
+  if ( !dumpFileName.empty() )
+  {
+    dumpFile.open(dumpFileName, std::ios::trunc);
```

Comment:
I would rather open the file further down an dremove the need for " if (dumpFile.is_open())"... or maybe this is not possible because the latter is within a loop while the "open" call is only once?